phoenix-core/src/main/java/org/apache/phoenix/query/QueryServicesOptions.java

```diff
@@ -354,6 +354,9 @@
     public static final long DEFAULT_TASK_HANDLING_MAX_INTERVAL_MS = 30*60*1000; // 30 min
     public static final long DEFAULT_TASK_HANDLING_INITIAL_DELAY_MS = 10*1000; // 10 sec
 
+    public static final long DEFAULT_GLOBAL_INDEX_ROW_AGE_THRESHOLD_TO_DELETE_MS = 10*60*1000; /* 10 min */
+    public static final int DEFAULT_GLOBAL_INDEX_REPAIR_COUNT = 100;
```

Comment:
Isn't 100 a bit low?

Reply:
It is the same value as DEFAULT_MUTATE_BATCH_SIZE. To make it clear, I will define as

public static final int DEFAULT_GLOBAL_INDEX_REPAIR_COUNT = DEFAULT_MUTATE_BATCH_SIZE;

Reply:
Done